### PR TITLE
Docstring parsing improvements

### LINKED ISSUE #1198: text parsing model creation doesn't work for vector variables
This would be great to have!

### PR DESCRIPTION
**Pull request by bqpd**
Closes #1198

### CODE REVIEW

File: nomen.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Lines that end in "Variables of length $N" will be parsed as vector
+    variables of length $N until the next blank line.
+
+    Variables of length 3
+    ---------------------
+    s       [m]    side length
+
+    The above variables are sufficient, but let's introduce more anyway:
+
+    Other Variables
```
Comment (mjburton11) [inline nomen.py]: Other is confusing.  * Variables

**Comment by bqpd** [comment]
note to self: make 
```
  """
    Variables
    ---------
    m    [kg ]  Motor mass
    W    [lbf]  Motor weight
    Pmax [W  ]  Max power
    """
```
throw a meaningful error

**Comment by mjburton11** [comment]
why does that throw an error?

**Comment by bqpd** [comment]
spaces in the unit declarations!

**Comment by mjburton11** [comment]
why is that a bad thing?

**Comment by mjburton11** [comment]
Ok I tried this for the first time convexengineering/gplibrary@9d78070b6638e9f420c9abd5bd51260033de4cb3 and I really like it.

**Comment by mjburton11** [comment]
I want to get this merged

**Comment by mjburton11** [comment]
Ok I have a problem on convexengineering/gplibrary@c1b7c9b94e585d9595687434c867f3c40ff6b415, the functions in the docstring don't seem to be working. Running `wing_test.py`

```python
/Users/mjburton11/MIT/GPKIT/library/gpkitmodels/GP/aircraft/wing/wing.py in return_c(self, c)
     44         " return normalized chord distribution "
     45         return np.array([2./(1+c[self.lam])*(1+(c[self.lam]-1)*e) for e
---> 46                          in c[self.eta]])
     47 
     48     def return_cmac(self, c):

AttributeError: 'Planform' object has no attribute 'eta'
```

**Comment by bqpd** [comment]
@mjburton11 the problem was the spaces in `np.linspace(0,1,5)` in the declaration of eta

**Comment by bqpd** [comment]
I used a `print parse_variables(Planform.__doc__)` line to debug

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
@mjburton11 it's now on master

**Pull request merged**